compiler-core/src/parse/snapshots/gleam_core__parse__tests__type_angle_generics_usage_error.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: compiler-core/src/parse/tests.rs
+expression: "let list: List<Int> = []"
+---
+----- SOURCE CODE
+let list: List<Int> = []
+
+----- ERROR
+error: Syntax error
+  ┌─ /src/parse/error.gleam:1:15
+  │
+1 │ let list: List<Int> = []
+  │               ^ I was expecting `(` here.
+
+Instead of angle brackets, Gleam uses parentheses for generic types:
+
+    List(String)
+    Result(Int, Error)
```

Comment:
Could we instead show the exact type they were going to write, but with the correct syntax? So here it would be `List(Int)`